packages/x-components/src/components/result/base-result-image.vue

```diff
@@ -208,14 +208,13 @@
 
 <style lang="scss" scoped>
   .x-result-picture {
+    // the loaderStyles prop is positioning absolutely to this container
+    position: relative;
     min-width: 1px;
     min-height: 1px;
-    position: relative;
 
     &__image {
-      max-width: 100%;
```

Comment:
If I'm not wrong.... 'max-width: 100%' is the default value for image tags, so removed

Reply:
Yes, you're right @mnavarroespinosa . Thks!